plugins/stack-overflow/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@backstage/plugin-stack-overflow",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.8.8",
+    "@backstage/plugin-home": "^0.4.15",
+    "@backstage/core-plugin-api": "^0.6.0",
+    "@backstage/search-common": "^0.2.2",
+    "@backstage/config": "^0.1.13",
+    "@backstage/theme": "^0.2.14",
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.57",
+    "qs": "^6.10.3",
+    "cross-fetch": "^3.0.6",
```

Comment:
this one's at `^3.1.5` too but no biggie. There's a double down in devDependencies you can remove though.